lib/adapter/freshheads/CssAdapter.ts

```diff
@@ -87,23 +90,18 @@ export default class CssAdapter implements Adapter {
 
         webpackConfig.module.rules.push(rule);
 
-        next();
-
         if (this.config.sass.enabled) {
-            new SassLoaderAdapter(this.config.sass).apply(
-                webpackConfig,
-                builderConfig,
-                () => {}
-            );
+            builder.add(new SassLoaderAdapter(this.config.sass));
+
         }
 
         if (!this.checkMiniCssExtractPluginIsInWebpackConfig(webpackConfig)) {
-            new ExtractCssPluginAdapter().apply(
-                webpackConfig,
-                builderConfig,
-                () => {}
-            );
+            builder.add(new ExtractCssPluginAdapter());
         }
```

Comment:
Volgens mij moet deze dus juist wel onder de `next()`. Als hij niet in een andere adapter is toegevoegd, dan moet hij hier alsnog toegevoegd worden.

Reply:
ok ja dat was eerst wel vond het al verwarrend.